docs/govuk_paas.md

```diff
@@ -65,6 +65,10 @@ cf logs [APP_NAME] --recent
 ```
 cf env [APP_NAME]
 ```
+### Elevate privileges to include SpaceDeveloper role (prior to shelling into production)
+```
+cf set-space-role <your-email-address> dfe bat-prod SpaceDeveloper
+```
```

Comment:
worth mentioning that this could either be `bat-prod` or `bat-qa`, depending on what you're up to?